Week-1/html/exercise_1.2/music_player/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <title>Music Player</title>
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Lato" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" 
+  integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="./css/playerfooter.css" />
+  <link rel="stylesheet" href="./css/currentsong.css" />
+  <link rel="stylesheet" href="./css/playlist.css" />
+  <link rel="stylesheet" href="./css/sidebar.css" />
+</head>
+
+<body>
+  <main>
+    <article class='main-content display-flex'>
+
+      <article id="sidebar-wrapper" class="sidebar-wrapper">
+      </article>
+
+      <article class='song-playlist'>
+
+        <section class="album-wrapper">
+          <section class="album-image">
+            <img
+              src='https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg'
+              alt='album poster'></img>
+          </section>
+          <section class="album-details">
+            <section class="album-detail-wrapper">
+              <section class="album-name">
+                <h3>CattyBoard Top 100 Single Charts (11.06.36)</h3>
+              </section>
+              <section class="album-artist">
+                Unknown Artist
+              </section>
+              <section class="album-summary">
+                2016.Charts.100 songs
+              </section>
+            </section>
+
+            <section id="album-controls" class="album-controls">
+            </section>
+
+          </section>
+        </section>
+
+        <section id="playlist-wrapper" class="wrapper">
```

Comment:
Why empty tag ?, can we remove this tag ?

Reply:
the HTML elements are added through JS iterating over a data object